src/Chromely/ChromelyBasicApp.cs

```diff
@@ -15,7 +15,7 @@ namespace Chromely
     /// </summary>
     public class ChromelyBasicApp: ChromelyAppBase
     {
-        public sealed override void ConfigureCoreServices(ServiceCollection services)
+        public override void ConfigureCoreServices(ServiceCollection services)
```

Comment:
@smx-smx "sealed" here is intentional. 

ConfigureCoreServices is like a last resort. It registers core services that are not registered in ConfigureServices. This is why ConfigureServices is executed before ConfigureCoreServices and all services are registered with "TryAddSingleton" meaning those services will ONLY be registered if they are not previously registered. Changing that may give developers a "false" impression that "Core" services are registered separately. They are not - this is a safeguard.

All custom registration must be done in ConfigureServices. 

A more appropriate name would be ConfigureCoreServicesIfNotRegistered, but that could be noted in the method summary.

Reply:
Ah, that makes sense now. I reverted that change and indeed it still works if i do it inside `ConfigureServices`

Reply:
@smx-smx 

Great!

Do you want me to hold on on the merge based on the Linux issue you raised?